[2024-02-03 20:15:09,681: INFO: main: Welcom to our custom log]
[2024-02-04 17:32:24,221: INFO: main: Welcom to our custom log]
[2024-02-04 17:32:24,221: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-04 17:32:24,222: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-04 17:32:24,222: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-04 17:32:24,222: INFO: common: created directory at: artifacts]
[2024-02-04 17:32:24,222: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-04 17:32:24,222: ERROR: main: name 'os' is not defined]
Traceback (most recent call last):
  File "main.py", line 13, in <module>
    data_ingestion.main()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/stage_01_data_ingestion.py", line 16, in main
    data_ingestion.download_file()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/components/data_ingestion.py", line 30, in download_file
    raise e
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/components/data_ingestion.py", line 20, in download_file
    os.makedirs("artifacts/data_ingestion", exist_ok=True)
NameError: name 'os' is not defined
[2024-02-04 17:33:31,259: INFO: main: Welcom to our custom log]
[2024-02-04 17:33:31,259: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-04 17:33:31,260: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-04 17:33:31,260: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-04 17:33:31,260: INFO: common: created directory at: artifacts]
[2024-02-04 17:33:31,260: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-04 17:33:31,260: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/17iriTzXvzVbMtVrI1mNW0uZAwgqM9tE3/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-04 17:35:29,895: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/17iriTzXvzVbMtVrI1mNW0uZAwgqM9tE3/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-04 17:35:30,502: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-04 18:17:56,465: INFO: main: Welcom to our custom log]
[2024-02-04 18:17:56,465: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-04 18:17:56,466: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-04 18:17:56,466: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-04 18:17:56,466: INFO: common: created directory at: artifacts]
[2024-02-04 18:17:56,467: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-04 18:17:56,467: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/17iriTzXvzVbMtVrI1mNW0uZAwgqM9tE3/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-04 18:19:55,792: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/17iriTzXvzVbMtVrI1mNW0uZAwgqM9tE3/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-04 18:19:56,455: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-04 19:47:01,387: INFO: main: Welcom to our custom log]
[2024-02-04 19:47:01,388: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-04 19:47:01,389: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-04 19:47:01,389: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-04 19:47:01,389: INFO: common: created directory at: artifacts]
[2024-02-04 19:47:01,390: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-04 19:47:01,390: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/17iriTzXvzVbMtVrI1mNW0uZAwgqM9tE3/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-04 19:48:59,368: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/17iriTzXvzVbMtVrI1mNW0uZAwgqM9tE3/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-04 19:49:00,048: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-05 10:07:55,751: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 10:07:55,755: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 10:07:55,758: INFO: common: created directory at: artifacts]
[2024-02-05 10:09:08,970: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 10:09:08,972: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 10:09:08,973: INFO: common: created directory at: artifacts]
[2024-02-05 10:09:12,476: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 10:09:12,478: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 10:09:12,479: INFO: common: created directory at: artifacts]
[2024-02-05 10:10:39,113: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 10:10:39,115: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 10:10:39,116: INFO: common: created directory at: artifacts]
[2024-02-05 10:10:39,117: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-05 10:11:29,967: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-05 16:09:05,580: INFO: main: Welcom to our custom log]
[2024-02-05 16:09:05,581: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-05 16:09:05,582: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 16:09:05,583: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 16:09:05,583: INFO: common: created directory at: artifacts]
[2024-02-05 16:09:05,583: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-05 16:09:05,583: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/17iriTzXvzVbMtVrI1mNW0uZAwgqM9tE3/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-05 16:11:01,481: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/17iriTzXvzVbMtVrI1mNW0uZAwgqM9tE3/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-05 16:11:02,139: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-05 16:11:02,139: INFO: main: *******************]
[2024-02-05 16:11:02,140: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-05 16:11:02,140: ERROR: main: name 'PrepareBaseModelTrainingPipeline' is not defined]
Traceback (most recent call last):
  File "main.py", line 25, in <module>
    prepare_base_model = PrepareBaseModelTrainingPipeline()
NameError: name 'PrepareBaseModelTrainingPipeline' is not defined
[2024-02-05 16:12:31,095: INFO: main: Welcom to our custom log]
[2024-02-05 16:12:31,095: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-05 16:12:31,097: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 16:12:31,098: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 16:12:31,098: INFO: common: created directory at: artifacts]
[2024-02-05 16:12:31,098: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-05 16:12:31,098: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/17iriTzXvzVbMtVrI1mNW0uZAwgqM9tE3/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-05 16:14:31,054: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/17iriTzXvzVbMtVrI1mNW0uZAwgqM9tE3/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-05 16:14:31,736: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-05 16:14:31,736: INFO: main: *******************]
[2024-02-05 16:14:31,736: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-05 16:14:31,737: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 16:14:31,738: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 16:14:31,738: INFO: common: created directory at: artifacts]
[2024-02-05 16:14:31,738: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-05 16:14:32,083: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-05 16:14:32,348: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-05 17:00:50,749: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:00:50,751: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:00:50,752: INFO: common: created directory at: artifacts]
[2024-02-05 17:00:50,753: INFO: common: created directory at: artifacts/training]
[2024-02-05 17:03:07,817: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:03:07,819: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:03:07,820: INFO: common: created directory at: artifacts]
[2024-02-05 17:03:07,821: INFO: common: created directory at: artifacts/training]
[2024-02-05 17:05:04,104: INFO: main: Welcom to our custom log]
[2024-02-05 17:05:04,104: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-05 17:05:04,105: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:05:04,106: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:05:04,106: INFO: common: created directory at: artifacts]
[2024-02-05 17:05:04,106: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-05 17:05:04,106: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/17iriTzXvzVbMtVrI1mNW0uZAwgqM9tE3/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-05 17:07:00,952: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/17iriTzXvzVbMtVrI1mNW0uZAwgqM9tE3/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-05 17:07:01,571: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-05 17:07:01,571: INFO: main: *******************]
[2024-02-05 17:07:01,572: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-05 17:07:01,573: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:07:01,574: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:07:01,574: INFO: common: created directory at: artifacts]
[2024-02-05 17:07:01,574: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-05 17:07:01,850: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-05 17:07:02,000: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-05 17:08:42,141: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:08:42,143: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:08:42,143: INFO: common: created directory at: artifacts]
[2024-02-05 17:08:42,144: INFO: common: created directory at: artifacts/training]
[2024-02-05 17:17:35,329: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:17:35,332: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:17:35,333: INFO: common: created directory at: artifacts]
[2024-02-05 17:17:35,334: INFO: common: created directory at: artifacts/training]
[2024-02-05 17:23:08,379: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:23:08,382: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:23:08,383: INFO: common: created directory at: artifacts]
[2024-02-05 17:23:08,384: INFO: common: created directory at: artifacts/training]
[2024-02-05 17:27:42,564: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:27:42,567: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:27:42,569: INFO: common: created directory at: artifacts]
[2024-02-05 17:27:42,569: INFO: common: created directory at: artifacts/training]
[2024-02-05 17:31:28,294: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:31:28,296: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:31:28,296: INFO: common: created directory at: artifacts]
[2024-02-05 17:31:28,297: INFO: common: created directory at: artifacts/training]
[2024-02-05 17:32:13,965: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:32:13,969: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:32:13,970: INFO: common: created directory at: artifacts]
[2024-02-05 17:32:13,970: INFO: common: created directory at: artifacts/training]
[2024-02-05 17:36:17,520: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:36:17,522: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:36:17,523: INFO: common: created directory at: artifacts]
[2024-02-05 17:36:17,524: INFO: common: created directory at: artifacts/training]
[2024-02-05 17:54:45,557: INFO: main: Welcom to our custom log]
[2024-02-05 17:54:45,557: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-05 17:54:45,558: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:54:45,559: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:54:45,560: INFO: common: created directory at: artifacts]
[2024-02-05 17:54:45,560: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-05 17:54:45,560: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-05 17:55:37,693: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-05 17:55:38,104: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-05 17:55:38,104: INFO: main: *******************]
[2024-02-05 17:55:38,104: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-05 17:55:38,105: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:55:38,107: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:55:38,107: INFO: common: created directory at: artifacts]
[2024-02-05 17:55:38,107: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-05 17:55:38,574: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-05 17:55:38,770: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-05 17:55:38,770: INFO: main: *******************]
[2024-02-05 17:55:38,770: INFO: main: >>>>>> stage training started <<<<<<]
[2024-02-05 17:55:38,771: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 17:55:38,772: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 17:55:38,772: INFO: common: created directory at: artifacts]
[2024-02-05 17:55:38,773: INFO: common: created directory at: artifacts/training]
[2024-02-05 17:55:39,037: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/kidney-CT-scan-image']
Traceback (most recent call last):
  File "main.py", line 38, in <module>
    prepare_base_model.main()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/stage_03_model_training.py", line 20, in main
    training.train_valid_generator()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/components/model_training.py", line 37, in train_valid_generator
    self.valid_generator = valid_datagenerator.flow_from_directory(
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/preprocessing/image.py", line 1648, in flow_from_directory
    return DirectoryIterator(
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/preprocessing/image.py", line 563, in __init__
    for subdir in sorted(os.listdir(directory)):
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/kidney-CT-scan-image'
[2024-02-05 18:00:33,550: INFO: main: Welcom to our custom log]
[2024-02-05 18:00:33,550: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-05 18:00:33,551: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 18:00:33,552: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 18:00:33,553: INFO: common: created directory at: artifacts]
[2024-02-05 18:00:33,553: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-05 18:00:33,553: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-05 18:01:25,977: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-05 18:01:26,415: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-05 18:01:26,415: INFO: main: *******************]
[2024-02-05 18:01:26,415: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-05 18:01:26,417: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 18:01:26,419: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 18:01:26,420: INFO: common: created directory at: artifacts]
[2024-02-05 18:01:26,420: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-05 18:01:26,914: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-05 18:01:27,202: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-05 18:01:27,203: INFO: main: *******************]
[2024-02-05 18:01:27,203: INFO: main: >>>>>> stage training started <<<<<<]
[2024-02-05 18:01:27,206: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 18:01:27,207: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 18:01:27,207: INFO: common: created directory at: artifacts]
[2024-02-05 18:01:27,208: INFO: common: created directory at: artifacts/training]
[2024-02-05 18:01:27,541: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/kidney-CT-scan-image']
Traceback (most recent call last):
  File "main.py", line 38, in <module>
    prepare_base_model.main()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/stage_03_model_training.py", line 20, in main
    training.train_valid_generator()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/components/model_training.py", line 37, in train_valid_generator
    self.valid_generator = valid_datagenerator.flow_from_directory(
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/preprocessing/image.py", line 1648, in flow_from_directory
    return DirectoryIterator(
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/preprocessing/image.py", line 563, in __init__
    for subdir in sorted(os.listdir(directory)):
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/kidney-CT-scan-image'
[2024-02-05 18:04:21,008: INFO: main: Welcom to our custom log]
[2024-02-05 18:04:21,008: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-05 18:04:21,010: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 18:04:21,011: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 18:04:21,011: INFO: common: created directory at: artifacts]
[2024-02-05 18:04:21,011: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-05 18:04:21,011: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-05 18:05:11,384: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-05 18:05:11,657: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-05 18:05:11,657: INFO: main: *******************]
[2024-02-05 18:05:11,657: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-05 18:05:11,659: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 18:05:11,659: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 18:05:11,660: INFO: common: created directory at: artifacts]
[2024-02-05 18:05:11,660: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-05 18:05:11,950: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-05 18:05:12,130: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-05 18:05:12,130: INFO: main: *******************]
[2024-02-05 18:05:12,130: INFO: main: >>>>>> stage training started <<<<<<]
[2024-02-05 18:05:12,131: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 18:05:12,132: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 18:05:12,132: INFO: common: created directory at: artifacts]
[2024-02-05 18:05:12,132: INFO: common: created directory at: artifacts/training]
[2024-02-05 18:05:52,891: INFO: main: >>>>>> stage training completed <<<<<<

x==========x]
[2024-02-05 23:29:10,782: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-05 23:29:10,787: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-05 23:29:10,788: INFO: common: created directory at: artifacts]
[2024-02-05 23:29:22,357: INFO: common: json file saved at: scores.json]
[2024-02-05 23:29:25,821: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-05 23:29:26,233: INFO: builder_impl: Assets written to: /tmp/tmpas0tsrlf/model/data/model/assets]
[2024-02-06 20:15:41,543: INFO: main: Welcom to our custom log]
[2024-02-06 20:15:41,543: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-06 20:15:41,545: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:15:41,546: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:15:41,546: INFO: common: created directory at: artifacts]
[2024-02-06 20:15:41,546: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-06 20:15:41,546: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:16:33,142: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:16:33,427: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-06 20:16:33,427: INFO: main: *******************]
[2024-02-06 20:16:33,427: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-06 20:16:33,429: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:16:33,429: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:16:33,430: INFO: common: created directory at: artifacts]
[2024-02-06 20:16:33,430: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-06 20:16:33,700: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-06 20:16:33,906: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-06 20:16:33,906: INFO: main: *******************]
[2024-02-06 20:16:33,906: INFO: main: >>>>>> stage training started <<<<<<]
[2024-02-06 20:16:33,907: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:16:33,908: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:16:33,908: INFO: common: created directory at: artifacts]
[2024-02-06 20:16:33,908: INFO: common: created directory at: artifacts/training]
[2024-02-06 20:17:29,436: INFO: main: >>>>>> stage training completed <<<<<<

x==========x]
[2024-02-06 20:17:29,436: INFO: main: *******************]
[2024-02-06 20:17:29,436: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-06 20:17:29,438: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:17:29,438: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:17:29,439: INFO: common: created directory at: artifacts]
[2024-02-06 20:17:47,312: INFO: common: json file saved at: scores.json]
[2024-02-06 20:17:47,314: INFO: common: json file saved at: scores.json]
[2024-02-06 20:17:47,314: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-06 20:19:33,695: INFO: main: Welcom to our custom log]
[2024-02-06 20:19:33,695: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-06 20:19:33,697: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:19:33,698: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:19:33,698: INFO: common: created directory at: artifacts]
[2024-02-06 20:19:33,698: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-06 20:19:33,698: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:20:25,786: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:20:26,077: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-06 20:20:26,077: INFO: main: *******************]
[2024-02-06 20:20:26,077: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-06 20:20:26,078: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:20:26,079: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:20:26,079: INFO: common: created directory at: artifacts]
[2024-02-06 20:20:26,079: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-06 20:20:26,346: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-06 20:20:26,536: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-06 20:20:26,536: INFO: main: *******************]
[2024-02-06 20:20:26,536: INFO: main: >>>>>> stage training started <<<<<<]
[2024-02-06 20:20:26,538: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:20:26,539: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:20:26,539: INFO: common: created directory at: artifacts]
[2024-02-06 20:20:26,539: INFO: common: created directory at: artifacts/training]
[2024-02-06 20:21:22,412: INFO: main: >>>>>> stage training completed <<<<<<

x==========x]
[2024-02-06 20:21:22,412: INFO: main: *******************]
[2024-02-06 20:21:22,412: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-06 20:21:22,414: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:21:22,415: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:21:22,415: INFO: common: created directory at: artifacts]
[2024-02-06 20:21:40,155: INFO: common: json file saved at: scores.json]
[2024-02-06 20:21:40,155: INFO: common: json file saved at: scores.json]
[2024-02-06 20:21:43,512: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-06 20:21:43,871: INFO: builder_impl: Assets written to: /tmp/tmptu6wuv85/model/data/model/assets]
[2024-02-06 20:24:34,002: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-06 20:25:05,757: INFO: main: Welcom to our custom log]
[2024-02-06 20:25:05,757: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-06 20:25:05,759: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:25:05,760: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:25:05,760: INFO: common: created directory at: artifacts]
[2024-02-06 20:25:05,760: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-06 20:25:05,760: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:25:57,266: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:25:57,566: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-06 20:25:57,566: INFO: main: *******************]
[2024-02-06 20:25:57,566: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-06 20:25:57,567: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:25:57,568: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:25:57,568: INFO: common: created directory at: artifacts]
[2024-02-06 20:25:57,568: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-06 20:25:57,830: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-06 20:25:58,012: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-06 20:25:58,012: INFO: main: *******************]
[2024-02-06 20:25:58,012: INFO: main: >>>>>> stage training started <<<<<<]
[2024-02-06 20:25:58,013: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:25:58,014: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:25:58,015: INFO: common: created directory at: artifacts]
[2024-02-06 20:25:58,015: INFO: common: created directory at: artifacts/training]
[2024-02-06 20:26:00,948: ERROR: main: Graph execution error:

Detected at node 'categorical_crossentropy/softmax_cross_entropy_with_logits' defined at (most recent call last):
    File "main.py", line 41, in <module>
      prepare_base_model.main()
    File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/stage_03_model_training.py", line 21, in main
      training.train()
    File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/components/model_training.py", line 76, in train
      self.model.fit(
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 65, in error_handler
      return fn(*args, **kwargs)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1685, in fit
      tmp_logs = self.train_function(iterator)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1284, in train_function
      return step_function(self, iterator)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1268, in step_function
      outputs = model.distribute_strategy.run(run_step, args=(data,))
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1249, in run_step
      outputs = model.train_step(data)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1051, in train_step
      loss = self.compute_loss(x, y, y_pred, sample_weight)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1109, in compute_loss
      return self.compiled_loss(
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/compile_utils.py", line 265, in __call__
      loss_value = loss_obj(y_t, y_p, sample_weight=sw)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/losses.py", line 142, in __call__
      losses = call_fn(y_true, y_pred)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/losses.py", line 268, in call
      return ag_fn(y_true, y_pred, **self._fn_kwargs)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/losses.py", line 1984, in categorical_crossentropy
      return backend.categorical_crossentropy(
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/backend.py", line 5565, in categorical_crossentropy
      return tf.nn.softmax_cross_entropy_with_logits(
Node: 'categorical_crossentropy/softmax_cross_entropy_with_logits'
logits and labels must be broadcastable: logits_size=[16,3] labels_size=[16,2]
	 [[{{node categorical_crossentropy/softmax_cross_entropy_with_logits}}]] [Op:__inference_train_function_1463]]
Traceback (most recent call last):
  File "main.py", line 41, in <module>
    prepare_base_model.main()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/stage_03_model_training.py", line 21, in main
    training.train()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/components/model_training.py", line 76, in train
    self.model.fit(
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/tensorflow/python/eager/execute.py", line 52, in quick_execute
    tensors = pywrap_tfe.TFE_Py_Execute(ctx._handle, device_name, op_name,
tensorflow.python.framework.errors_impl.InvalidArgumentError: Graph execution error:

Detected at node 'categorical_crossentropy/softmax_cross_entropy_with_logits' defined at (most recent call last):
    File "main.py", line 41, in <module>
      prepare_base_model.main()
    File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/stage_03_model_training.py", line 21, in main
      training.train()
    File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/components/model_training.py", line 76, in train
      self.model.fit(
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 65, in error_handler
      return fn(*args, **kwargs)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1685, in fit
      tmp_logs = self.train_function(iterator)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1284, in train_function
      return step_function(self, iterator)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1268, in step_function
      outputs = model.distribute_strategy.run(run_step, args=(data,))
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1249, in run_step
      outputs = model.train_step(data)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1051, in train_step
      loss = self.compute_loss(x, y, y_pred, sample_weight)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1109, in compute_loss
      return self.compiled_loss(
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/compile_utils.py", line 265, in __call__
      loss_value = loss_obj(y_t, y_p, sample_weight=sw)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/losses.py", line 142, in __call__
      losses = call_fn(y_true, y_pred)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/losses.py", line 268, in call
      return ag_fn(y_true, y_pred, **self._fn_kwargs)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/losses.py", line 1984, in categorical_crossentropy
      return backend.categorical_crossentropy(
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/backend.py", line 5565, in categorical_crossentropy
      return tf.nn.softmax_cross_entropy_with_logits(
Node: 'categorical_crossentropy/softmax_cross_entropy_with_logits'
logits and labels must be broadcastable: logits_size=[16,3] labels_size=[16,2]
	 [[{{node categorical_crossentropy/softmax_cross_entropy_with_logits}}]] [Op:__inference_train_function_1463]
[2024-02-06 20:26:42,081: INFO: main: Welcom to our custom log]
[2024-02-06 20:26:42,081: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-06 20:26:42,083: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:26:42,084: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:26:42,084: INFO: common: created directory at: artifacts]
[2024-02-06 20:26:42,084: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-06 20:26:42,084: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:27:33,188: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:27:33,480: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-06 20:27:33,480: INFO: main: *******************]
[2024-02-06 20:27:33,480: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-06 20:27:33,481: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:27:33,482: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:27:33,482: INFO: common: created directory at: artifacts]
[2024-02-06 20:27:33,482: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-06 20:27:33,746: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-06 20:27:33,933: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-06 20:27:33,933: INFO: main: *******************]
[2024-02-06 20:27:33,933: INFO: main: >>>>>> stage training started <<<<<<]
[2024-02-06 20:27:33,935: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:27:33,936: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:27:33,936: INFO: common: created directory at: artifacts]
[2024-02-06 20:27:33,936: INFO: common: created directory at: artifacts/training]
[2024-02-06 20:27:36,867: ERROR: main: Graph execution error:

Detected at node 'categorical_crossentropy/softmax_cross_entropy_with_logits' defined at (most recent call last):
    File "main.py", line 41, in <module>
      prepare_base_model.main()
    File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/stage_03_model_training.py", line 21, in main
      training.train()
    File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/components/model_training.py", line 76, in train
      self.model.fit(
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 65, in error_handler
      return fn(*args, **kwargs)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1685, in fit
      tmp_logs = self.train_function(iterator)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1284, in train_function
      return step_function(self, iterator)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1268, in step_function
      outputs = model.distribute_strategy.run(run_step, args=(data,))
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1249, in run_step
      outputs = model.train_step(data)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1051, in train_step
      loss = self.compute_loss(x, y, y_pred, sample_weight)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1109, in compute_loss
      return self.compiled_loss(
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/compile_utils.py", line 265, in __call__
      loss_value = loss_obj(y_t, y_p, sample_weight=sw)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/losses.py", line 142, in __call__
      losses = call_fn(y_true, y_pred)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/losses.py", line 268, in call
      return ag_fn(y_true, y_pred, **self._fn_kwargs)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/losses.py", line 1984, in categorical_crossentropy
      return backend.categorical_crossentropy(
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/backend.py", line 5565, in categorical_crossentropy
      return tf.nn.softmax_cross_entropy_with_logits(
Node: 'categorical_crossentropy/softmax_cross_entropy_with_logits'
logits and labels must be broadcastable: logits_size=[16,3] labels_size=[16,2]
	 [[{{node categorical_crossentropy/softmax_cross_entropy_with_logits}}]] [Op:__inference_train_function_1463]]
Traceback (most recent call last):
  File "main.py", line 41, in <module>
    prepare_base_model.main()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/stage_03_model_training.py", line 21, in main
    training.train()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/components/model_training.py", line 76, in train
    self.model.fit(
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/tensorflow/python/eager/execute.py", line 52, in quick_execute
    tensors = pywrap_tfe.TFE_Py_Execute(ctx._handle, device_name, op_name,
tensorflow.python.framework.errors_impl.InvalidArgumentError: Graph execution error:

Detected at node 'categorical_crossentropy/softmax_cross_entropy_with_logits' defined at (most recent call last):
    File "main.py", line 41, in <module>
      prepare_base_model.main()
    File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/stage_03_model_training.py", line 21, in main
      training.train()
    File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/components/model_training.py", line 76, in train
      self.model.fit(
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 65, in error_handler
      return fn(*args, **kwargs)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1685, in fit
      tmp_logs = self.train_function(iterator)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1284, in train_function
      return step_function(self, iterator)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1268, in step_function
      outputs = model.distribute_strategy.run(run_step, args=(data,))
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1249, in run_step
      outputs = model.train_step(data)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1051, in train_step
      loss = self.compute_loss(x, y, y_pred, sample_weight)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/training.py", line 1109, in compute_loss
      return self.compiled_loss(
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/engine/compile_utils.py", line 265, in __call__
      loss_value = loss_obj(y_t, y_p, sample_weight=sw)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/losses.py", line 142, in __call__
      losses = call_fn(y_true, y_pred)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/losses.py", line 268, in call
      return ag_fn(y_true, y_pred, **self._fn_kwargs)
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/losses.py", line 1984, in categorical_crossentropy
      return backend.categorical_crossentropy(
    File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/backend.py", line 5565, in categorical_crossentropy
      return tf.nn.softmax_cross_entropy_with_logits(
Node: 'categorical_crossentropy/softmax_cross_entropy_with_logits'
logits and labels must be broadcastable: logits_size=[16,3] labels_size=[16,2]
	 [[{{node categorical_crossentropy/softmax_cross_entropy_with_logits}}]] [Op:__inference_train_function_1463]
[2024-02-06 20:28:00,326: INFO: main: Welcom to our custom log]
[2024-02-06 20:28:00,326: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-06 20:28:00,327: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:28:00,328: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:28:00,328: INFO: common: created directory at: artifacts]
[2024-02-06 20:28:00,328: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-06 20:28:00,329: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:28:52,208: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:28:52,577: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-06 20:28:52,577: INFO: main: *******************]
[2024-02-06 20:28:52,577: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-06 20:28:52,578: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:28:52,579: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:28:52,579: INFO: common: created directory at: artifacts]
[2024-02-06 20:28:52,579: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-06 20:28:52,892: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-06 20:28:53,089: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-06 20:28:53,089: INFO: main: *******************]
[2024-02-06 20:28:53,089: INFO: main: >>>>>> stage training started <<<<<<]
[2024-02-06 20:28:53,090: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:28:53,091: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:28:53,091: INFO: common: created directory at: artifacts]
[2024-02-06 20:28:53,091: INFO: common: created directory at: artifacts/training]
[2024-02-06 20:29:48,808: INFO: main: >>>>>> stage training completed <<<<<<

x==========x]
[2024-02-06 20:29:48,808: INFO: main: *******************]
[2024-02-06 20:29:48,808: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-06 20:29:48,810: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:29:48,811: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:29:48,811: INFO: common: created directory at: artifacts]
[2024-02-06 20:30:06,418: INFO: common: json file saved at: scores.json]
[2024-02-06 20:30:06,420: INFO: common: json file saved at: scores.json]
[2024-02-06 20:30:09,678: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-06 20:30:10,001: INFO: builder_impl: Assets written to: /tmp/tmphumvqrtg/model/data/model/assets]
[2024-02-06 20:31:52,087: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-06 20:32:37,631: INFO: main: Welcom to our custom log]
[2024-02-06 20:32:37,631: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-06 20:32:37,633: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:32:37,633: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:32:37,634: INFO: common: created directory at: artifacts]
[2024-02-06 20:32:37,634: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-06 20:32:37,634: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:33:29,140: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:33:29,431: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-06 20:33:29,431: INFO: main: *******************]
[2024-02-06 20:33:29,431: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-06 20:33:29,432: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:33:29,433: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:33:29,433: INFO: common: created directory at: artifacts]
[2024-02-06 20:33:29,434: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-06 20:33:29,694: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-06 20:33:29,893: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-06 20:33:29,894: INFO: main: *******************]
[2024-02-06 20:33:29,894: INFO: main: >>>>>> stage training started <<<<<<]
[2024-02-06 20:33:29,895: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:33:29,896: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:33:29,896: INFO: common: created directory at: artifacts]
[2024-02-06 20:33:29,896: INFO: common: created directory at: artifacts/training]
[2024-02-06 20:35:06,182: INFO: main: >>>>>> stage training completed <<<<<<

x==========x]
[2024-02-06 20:35:06,183: INFO: main: *******************]
[2024-02-06 20:35:06,183: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-06 20:35:06,184: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:35:06,185: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:35:06,185: INFO: common: created directory at: artifacts]
[2024-02-06 20:35:16,986: INFO: common: json file saved at: scores.json]
[2024-02-06 20:35:16,988: INFO: common: json file saved at: scores.json]
[2024-02-06 20:35:20,334: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-06 20:35:20,687: INFO: builder_impl: Assets written to: /tmp/tmpfvertkz6/model/data/model/assets]
[2024-02-06 20:37:02,977: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-06 20:40:43,829: INFO: main: Welcom to our custom log]
[2024-02-06 20:40:43,829: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-06 20:40:43,831: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:40:43,832: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:40:43,832: INFO: common: created directory at: artifacts]
[2024-02-06 20:40:43,832: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-06 20:40:43,832: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:41:35,565: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 20:41:35,862: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-06 20:41:35,862: INFO: main: *******************]
[2024-02-06 20:41:35,862: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-06 20:41:35,863: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:41:35,864: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:41:35,864: INFO: common: created directory at: artifacts]
[2024-02-06 20:41:35,864: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-06 20:41:36,122: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-06 20:41:36,302: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-06 20:41:36,302: INFO: main: *******************]
[2024-02-06 20:41:36,302: INFO: main: >>>>>> stage training started <<<<<<]
[2024-02-06 20:41:36,303: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:41:36,304: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:41:36,304: INFO: common: created directory at: artifacts]
[2024-02-06 20:41:36,305: INFO: common: created directory at: artifacts/training]
[2024-02-06 20:42:07,666: INFO: main: >>>>>> stage training completed <<<<<<

x==========x]
[2024-02-06 20:42:07,666: INFO: main: *******************]
[2024-02-06 20:42:07,666: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-06 20:42:07,667: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 20:42:07,668: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 20:42:07,668: INFO: common: created directory at: artifacts]
[2024-02-06 20:42:18,023: INFO: common: json file saved at: scores.json]
[2024-02-06 20:42:18,023: INFO: common: json file saved at: scores.json]
[2024-02-06 20:42:21,078: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-06 20:42:21,428: INFO: builder_impl: Assets written to: /tmp/tmpfx769xwb/model/data/model/assets]
[2024-02-06 20:44:01,414: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-06 21:34:10,080: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-06 21:34:10,081: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 21:34:10,082: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 21:34:10,083: INFO: common: created directory at: artifacts]
[2024-02-06 21:34:10,083: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-06 21:34:10,083: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 21:35:10,113: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-06 21:35:10,387: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-06 21:35:12,571: INFO: stage_02_prepare_base_model: *******************]
[2024-02-06 21:35:12,571: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-06 21:35:12,573: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 21:35:12,574: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 21:35:12,574: INFO: common: created directory at: artifacts]
[2024-02-06 21:35:12,575: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-06 21:35:12,840: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-06 21:35:12,981: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-06 21:35:15,090: INFO: stage_03_model_training: *******************]
[2024-02-06 21:35:15,090: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-06 21:35:15,092: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 21:35:15,094: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 21:35:15,094: INFO: common: created directory at: artifacts]
[2024-02-06 21:35:15,094: INFO: common: created directory at: artifacts/training]
[2024-02-06 21:37:04,562: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-02-06 21:37:07,171: INFO: stage_04_model_evaluation: *******************]
[2024-02-06 21:37:07,171: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-06 21:37:07,173: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-06 21:37:07,174: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-06 21:37:07,174: INFO: common: created directory at: artifacts]
[2024-02-06 21:37:18,242: INFO: common: json file saved at: scores.json]
[2024-02-06 21:37:18,242: INFO: common: json file saved at: scores.json]
[2024-02-06 21:37:21,778: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-06 21:37:22,180: INFO: builder_impl: Assets written to: /tmp/tmpyhz_iwfp/model/data/model/assets]
[2024-02-06 21:39:05,156: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-07 09:42:04,758: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.88.123:8080]
[2024-02-07 09:42:04,758: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-07 09:42:11,429: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 09:42:11] "GET / HTTP/1.1" 200 -]
[2024-02-07 09:42:53,112: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/saving/saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/saving/legacy/save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at model/model.h5
[2024-02-07 09:42:53,117: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 09:42:53] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-02-07 09:44:29,734: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 09:44:29] "GET / HTTP/1.1" 200 -]
[2024-02-07 09:44:43,056: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/saving/saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/saving/legacy/save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at model/model.h5
[2024-02-07 09:44:43,056: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 09:44:43] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-02-07 09:49:48,543: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.88.123:5000]
[2024-02-07 09:49:48,543: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-07 09:49:52,117: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 09:49:52] "GET / HTTP/1.1" 200 -]
[2024-02-07 09:50:02,266: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/saving/saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/saving/legacy/save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at model/model.h5
[2024-02-07 09:50:02,267: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 09:50:02] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-02-07 09:53:50,695: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.88.123:5000]
[2024-02-07 09:53:50,695: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-07 09:53:55,270: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 09:53:55] "GET / HTTP/1.1" 200 -]
[2024-02-07 09:54:01,234: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "/home/fintechsys/MLflow projects/kidney-Disease-classification-Deep-Learning--project/src/cnnClassifier/pipeline/prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/saving/saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/home/fintechsys/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/saving/legacy/save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at model/model.h5
[2024-02-07 09:54:01,236: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 09:54:01] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-02-07 09:57:33,041: INFO: stage_03_model_training: *******************]
[2024-02-07 09:57:33,041: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-07 09:57:33,045: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-07 09:57:33,046: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-07 09:57:33,046: INFO: common: created directory at: artifacts]
[2024-02-07 09:57:33,047: INFO: common: created directory at: artifacts/training]
[2024-02-07 09:58:29,243: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-02-07 09:58:32,208: INFO: stage_04_model_evaluation: *******************]
[2024-02-07 09:58:32,209: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-07 09:58:32,210: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-07 09:58:32,211: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-07 09:58:32,211: INFO: common: created directory at: artifacts]
[2024-02-07 09:58:49,968: INFO: common: json file saved at: scores.json]
[2024-02-07 09:58:49,968: INFO: common: json file saved at: scores.json]
[2024-02-07 09:58:52,772: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-07 09:58:53,138: INFO: builder_impl: Assets written to: /tmp/tmpq3_pgj4c/model/data/model/assets]
[2024-02-07 10:05:07,734: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-07 12:10:09,350: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.88.123:5001]
[2024-02-07 12:10:09,350: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-07 12:10:12,805: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 12:10:12] "GET / HTTP/1.1" 200 -]
[2024-02-07 12:10:22,315: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 12:10:22] "POST /predict HTTP/1.1" 200 -]
[2024-02-07 12:10:37,317: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 12:10:37] "POST /predict HTTP/1.1" 200 -]
[2024-02-07 12:10:39,859: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 12:10:39] "POST /predict HTTP/1.1" 200 -]
[2024-02-07 12:33:08,785: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.88.123:5001]
[2024-02-07 12:33:08,786: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-07 12:33:11,956: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 12:33:11] "GET / HTTP/1.1" 200 -]
[2024-02-07 12:33:24,336: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 12:33:24] "POST /predict HTTP/1.1" 200 -]
[2024-02-07 12:34:10,479: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 12:34:10] "GET /train HTTP/1.1" 200 -]
[2024-02-07 12:34:10,518: INFO: _internal: 127.0.0.1 - - [07/Feb/2024 12:34:10] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-02-08 11:54:28,987: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-08 11:54:28,989: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-08 11:54:28,991: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-08 11:54:28,991: INFO: common: created directory at: artifacts]
[2024-02-08 11:54:28,991: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-08 11:54:28,991: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1ySHkBbc20UQRGh3ZwYfHZRc_G8-2ak8E/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-08 11:58:41,742: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-08 11:58:41,743: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-08 11:58:41,744: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-08 11:58:41,745: INFO: common: created directory at: artifacts]
[2024-02-08 11:58:41,745: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-08 11:58:41,745: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1ySHkBbc20UQRGh3ZwYfHZRc_G8-2ak8E/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-08 12:01:38,872: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1ySHkBbc20UQRGh3ZwYfHZRc_G8-2ak8E/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-08 12:01:39,543: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-08 12:07:33,766: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-08 12:07:33,767: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-08 12:07:33,768: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-08 12:07:33,768: INFO: common: created directory at: artifacts]
[2024-02-08 12:07:33,768: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-08 12:07:33,768: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1ySHkBbc20UQRGh3ZwYfHZRc_G8-2ak8E/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-08 12:11:15,049: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1ySHkBbc20UQRGh3ZwYfHZRc_G8-2ak8E/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-08 12:11:15,753: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-10 09:09:37,775: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-10 09:09:37,777: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 09:09:37,778: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 09:09:37,778: INFO: common: created directory at: artifacts]
[2024-02-10 09:09:37,778: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-10 09:09:37,778: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1ySHkBbc20UQRGh3ZwYfHZRc_G8-2ak8E/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-10 09:12:52,671: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1ySHkBbc20UQRGh3ZwYfHZRc_G8-2ak8E/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-10 09:12:53,343: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-10 09:22:06,321: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-10 09:22:06,322: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 09:22:06,323: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 09:22:06,323: INFO: common: created directory at: artifacts]
[2024-02-10 09:22:06,323: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-10 09:22:06,324: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1ySHkBbc20UQRGh3ZwYfHZRc_G8-2ak8E/view?usp=sharing into file artifacts/data_ingestion/data.zip]
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         [2024-02-10 09:30:12,444: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-10 09:30:12,446: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 09:30:12,447: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 09:30:12,447: INFO: common: created directory at: artifacts]
[2024-02-10 09:30:12,447: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-10 09:30:12,447: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1ySHkBbc20UQRGh3ZwYfHZRc_G8-2ak8E/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-10 09:32:59,123: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1ySHkBbc20UQRGh3ZwYfHZRc_G8-2ak8E/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-10 09:32:59,763: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-10 09:33:06,270: INFO: stage_02_prepare_base_model: *******************]
[2024-02-10 09:33:06,270: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-10 09:33:06,272: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 09:33:06,273: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 09:33:06,273: INFO: common: created directory at: artifacts]
[2024-02-10 09:33:06,273: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-10 09:33:06,589: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-10 09:33:06,734: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-10 09:33:14,216: INFO: stage_03_model_training: *******************]
[2024-02-10 09:33:14,216: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-10 09:33:14,218: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 09:33:14,219: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 09:33:14,219: INFO: common: created directory at: artifacts]
[2024-02-10 09:33:14,219: INFO: common: created directory at: artifacts/training]
[2024-02-10 09:38:00,925: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-02-10 09:38:07,429: INFO: stage_04_model_evaluation: *******************]
[2024-02-10 09:38:07,429: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-10 09:38:07,430: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 09:38:07,432: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 09:38:07,432: INFO: common: created directory at: artifacts]
[2024-02-10 09:39:25,610: INFO: common: json file saved at: scores.json]
[2024-02-10 09:39:25,610: INFO: common: json file saved at: scores.json]
[2024-02-10 09:39:28,675: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-10 09:39:29,084: INFO: builder_impl: Assets written to: /tmp/tmp3heghfzq/model/data/model/assets]
[2024-02-10 09:50:44,562: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-10 11:17:40,140: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-10 11:17:40,141: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 11:17:40,142: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 11:17:40,142: INFO: common: created directory at: artifacts]
[2024-02-10 11:17:40,143: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-10 11:17:40,143: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kbJ_Kpp2TqVVXzoy-Ju_QNFJ5tpt6FAu/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-10 11:21:12,582: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1kbJ_Kpp2TqVVXzoy-Ju_QNFJ5tpt6FAu/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-10 11:21:13,332: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-10 11:21:20,003: INFO: stage_02_prepare_base_model: *******************]
[2024-02-10 11:21:20,003: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-10 11:21:20,005: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 11:21:20,006: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 11:21:20,006: INFO: common: created directory at: artifacts]
[2024-02-10 11:21:20,006: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-10 11:21:20,300: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-10 11:21:20,451: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-10 11:21:23,592: INFO: stage_03_model_training: *******************]
[2024-02-10 11:21:23,592: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-10 11:21:23,594: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 11:21:23,595: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 11:21:23,595: INFO: common: created directory at: artifacts]
[2024-02-10 11:21:23,595: INFO: common: created directory at: artifacts/training]
[2024-02-10 11:26:10,827: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-02-10 11:26:14,966: INFO: stage_04_model_evaluation: *******************]
[2024-02-10 11:26:14,966: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-10 11:26:14,968: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 11:26:14,968: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 11:26:14,969: INFO: common: created directory at: artifacts]
[2024-02-10 11:27:43,189: INFO: common: json file saved at: scores.json]
[2024-02-10 11:27:43,191: INFO: common: json file saved at: scores.json]
[2024-02-10 11:27:46,069: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-10 11:27:46,457: INFO: builder_impl: Assets written to: /tmp/tmpdmg9tuax/model/data/model/assets]
[2024-02-10 11:39:37,913: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-10 11:43:38,349: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.88.123:5001]
[2024-02-10 11:43:38,349: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-10 11:43:41,705: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 11:43:41] "GET / HTTP/1.1" 200 -]
[2024-02-10 11:43:54,266: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 11:43:54] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 11:44:07,298: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 11:44:07] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 11:44:18,405: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 11:44:18] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 11:47:47,486: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 11:47:47] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 11:48:23,013: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x7fa3f05c7310> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-02-10 11:48:23,015: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 11:48:23] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 11:48:34,570: WARNING: polymorphic_function: 6 out of the last 6 calls to <function Model.make_predict_function.<locals>.predict_function at 0x7fa3f05c7af0> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-02-10 11:48:34,572: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 11:48:34] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 11:48:36,539: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 11:48:36] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 11:49:57,572: INFO: stage_03_model_training: *******************]
[2024-02-10 11:49:57,572: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-10 11:49:57,574: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 11:49:57,575: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 11:49:57,575: INFO: common: created directory at: artifacts]
[2024-02-10 11:49:57,575: INFO: common: created directory at: artifacts/training]
[2024-02-10 12:38:07,921: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-02-10 12:38:12,520: INFO: stage_04_model_evaluation: *******************]
[2024-02-10 12:38:12,520: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-10 12:38:12,522: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 12:38:12,523: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 12:38:12,523: INFO: common: created directory at: artifacts]
[2024-02-10 12:39:39,129: INFO: common: json file saved at: scores.json]
[2024-02-10 12:39:39,129: INFO: common: json file saved at: scores.json]
[2024-02-10 12:39:42,078: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-10 12:39:42,462: INFO: builder_impl: Assets written to: /tmp/tmpi_m6buhg/model/data/model/assets]
[2024-02-10 12:49:18,786: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-10 14:51:21,389: INFO: stage_03_model_training: *******************]
[2024-02-10 14:51:21,389: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-10 14:51:21,391: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 14:51:21,392: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 14:51:21,392: INFO: common: created directory at: artifacts]
[2024-02-10 14:51:21,392: INFO: common: created directory at: artifacts/training]
[2024-02-10 16:06:45,489: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-02-10 16:06:49,821: INFO: stage_04_model_evaluation: *******************]
[2024-02-10 16:06:49,821: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-10 16:06:49,823: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 16:06:49,824: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 16:06:49,824: INFO: common: created directory at: artifacts]
[2024-02-10 16:08:17,961: INFO: common: json file saved at: scores.json]
[2024-02-10 16:08:17,961: INFO: common: json file saved at: scores.json]
[2024-02-10 16:08:22,044: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-10 16:08:22,427: INFO: builder_impl: Assets written to: /tmp/tmpqcm59egb/model/data/model/assets]
[2024-02-10 16:10:02,578: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-10 16:23:51,177: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.88.54:8000]
[2024-02-10 16:23:51,177: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-10 16:23:55,220: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 16:23:55] "GET / HTTP/1.1" 200 -]
[2024-02-10 16:24:11,324: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 16:24:11] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 16:24:21,380: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 16:24:21] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 16:24:22,745: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 16:24:22] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 16:24:35,302: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 16:24:35] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 16:29:40,150: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x7f4c6010f160> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-02-10 16:29:40,151: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 16:29:40] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 16:29:57,243: WARNING: polymorphic_function: 6 out of the last 6 calls to <function Model.make_predict_function.<locals>.predict_function at 0x7f4c6010fa60> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-02-10 16:29:57,244: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 16:29:57] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 16:30:08,336: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 16:30:08] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 16:30:21,680: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 16:30:21] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 16:33:06,602: INFO: stage_02_prepare_base_model: *******************]
[2024-02-10 16:33:06,602: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-10 16:33:06,604: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 16:33:06,605: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 16:33:06,605: INFO: common: created directory at: artifacts]
[2024-02-10 16:33:06,606: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-10 16:33:06,917: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-10 16:33:07,060: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-10 16:33:10,503: INFO: stage_03_model_training: *******************]
[2024-02-10 16:33:10,503: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-10 16:33:10,505: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 16:33:10,506: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 16:33:10,507: INFO: common: created directory at: artifacts]
[2024-02-10 16:33:10,507: INFO: common: created directory at: artifacts/training]
[2024-02-10 17:37:10,640: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-02-10 17:37:15,150: INFO: stage_04_model_evaluation: *******************]
[2024-02-10 17:37:15,150: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-10 17:37:15,151: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 17:37:15,152: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 17:37:15,152: INFO: common: created directory at: artifacts]
[2024-02-10 17:39:11,291: INFO: common: json file saved at: scores.json]
[2024-02-10 17:39:11,291: INFO: common: json file saved at: scores.json]
[2024-02-10 17:39:14,020: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-10 17:39:14,397: INFO: builder_impl: Assets written to: /tmp/tmp1yddm9qr/model/data/model/assets]
[2024-02-10 17:40:56,597: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-10 17:41:18,368: INFO: stage_02_prepare_base_model: *******************]
[2024-02-10 17:41:18,368: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-10 17:41:18,370: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 17:41:18,371: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 17:41:18,372: INFO: common: created directory at: artifacts]
[2024-02-10 17:41:18,372: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-10 17:41:18,689: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-10 17:41:18,842: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-10 17:41:22,287: INFO: stage_03_model_training: *******************]
[2024-02-10 17:41:22,287: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-10 17:41:22,289: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 17:41:22,290: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 17:41:22,290: INFO: common: created directory at: artifacts]
[2024-02-10 17:41:22,290: INFO: common: created directory at: artifacts/training]
[2024-02-10 18:29:52,395: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-02-10 18:29:56,900: INFO: stage_04_model_evaluation: *******************]
[2024-02-10 18:29:56,900: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-10 18:29:56,902: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 18:29:56,903: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 18:29:56,903: INFO: common: created directory at: artifacts]
[2024-02-10 18:31:24,693: INFO: common: json file saved at: scores.json]
[2024-02-10 18:31:24,693: INFO: common: json file saved at: scores.json]
[2024-02-10 18:31:27,796: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-10 18:31:28,160: INFO: builder_impl: Assets written to: /tmp/tmpc4ljdx5x/model/data/model/assets]
[2024-02-10 18:33:10,067: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-10 18:33:25,774: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.88.54:8000]
[2024-02-10 18:33:25,774: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-10 18:33:29,696: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 18:33:29] "GET / HTTP/1.1" 200 -]
[2024-02-10 18:33:49,897: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 18:33:49] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 18:34:05,206: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 18:34:05] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 18:35:57,990: INFO: stage_03_model_training: *******************]
[2024-02-10 18:35:57,990: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-10 18:35:57,992: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 18:35:57,993: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 18:35:57,993: INFO: common: created directory at: artifacts]
[2024-02-10 18:35:57,994: INFO: common: created directory at: artifacts/training]
[2024-02-10 19:53:44,422: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-02-10 19:53:49,322: INFO: stage_04_model_evaluation: *******************]
[2024-02-10 19:53:49,322: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-10 19:53:49,324: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-10 19:53:49,325: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-10 19:53:49,325: INFO: common: created directory at: artifacts]
[2024-02-10 19:55:30,252: INFO: common: json file saved at: scores.json]
[2024-02-10 19:55:30,253: INFO: common: json file saved at: scores.json]
[2024-02-10 19:55:33,687: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-10 19:55:34,076: INFO: builder_impl: Assets written to: /tmp/tmpr_4yot4o/model/data/model/assets]
[2024-02-10 19:57:12,577: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-10 19:58:42,408: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8001
 * Running on http://192.168.88.54:8001]
[2024-02-10 19:58:42,408: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-10 19:58:46,661: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 19:58:46] "GET / HTTP/1.1" 200 -]
[2024-02-10 19:58:47,671: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 19:58:47] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-02-10 21:37:37,101: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8001
 * Running on http://192.168.1.115:8001]
[2024-02-10 21:37:37,101: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-10 21:37:45,567: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:37:45] "GET / HTTP/1.1" 200 -]
[2024-02-10 21:38:03,935: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:38:03] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:38:12,386: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:38:12] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:38:34,088: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:38:34] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:38:49,049: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:38:49] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:39:00,420: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x7f702c7ef4c0> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-02-10 21:39:00,422: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:39:00] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:39:16,025: WARNING: polymorphic_function: 6 out of the last 6 calls to <function Model.make_predict_function.<locals>.predict_function at 0x7f702c7ef790> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-02-10 21:39:16,028: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:39:16] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:39:26,956: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:39:26] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:39:45,041: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:39:45] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:41:30,472: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:41:30] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:41:47,852: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:41:47] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:42:04,312: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:42:04] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:42:05,638: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:42:05] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:42:10,112: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:42:10] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:42:12,353: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:42:12] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:42:18,819: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:42:18] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:42:23,948: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:42:23] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:42:28,525: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:42:28] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:43:18,493: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:43:18] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:43:33,703: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:43:33] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:43:42,390: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:43:42] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:43:44,572: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:43:44] "POST /predict HTTP/1.1" 200 -]
[2024-02-10 21:43:57,054: INFO: _internal: 127.0.0.1 - - [10/Feb/2024 21:43:57] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 08:38:22,510: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-11 08:38:22,512: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 08:38:22,513: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 08:38:22,513: INFO: common: created directory at: artifacts]
[2024-02-11 08:38:22,513: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-11 08:38:22,513: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kbJ_Kpp2TqVVXzoy-Ju_QNFJ5tpt6FAu/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-11 08:41:42,249: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1kbJ_Kpp2TqVVXzoy-Ju_QNFJ5tpt6FAu/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-11 08:41:42,937: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-11 08:41:48,854: INFO: stage_02_prepare_base_model: *******************]
[2024-02-11 08:41:48,855: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-11 08:41:48,857: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 08:41:48,858: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 08:41:48,858: INFO: common: created directory at: artifacts]
[2024-02-11 08:41:48,858: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-11 08:41:49,164: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-11 08:41:49,324: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-11 08:41:52,698: INFO: stage_03_model_training: *******************]
[2024-02-11 08:41:52,698: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-11 08:41:52,700: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 08:41:52,701: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 08:41:52,701: INFO: common: created directory at: artifacts]
[2024-02-11 08:41:52,701: INFO: common: created directory at: artifacts/training]
[2024-02-11 09:51:42,899: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-02-11 09:51:47,533: INFO: stage_04_model_evaluation: *******************]
[2024-02-11 09:51:47,533: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-11 09:51:47,534: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 09:51:47,535: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 09:51:47,535: INFO: common: created directory at: artifacts]
[2024-02-11 09:53:19,976: INFO: common: json file saved at: scores.json]
[2024-02-11 09:53:19,976: INFO: common: json file saved at: scores.json]
[2024-02-11 09:53:23,067: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-11 09:53:23,467: INFO: builder_impl: Assets written to: /tmp/tmp5ses3c7s/model/data/model/assets]
[2024-02-11 09:59:34,803: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-11 10:09:25,035: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.88.54:8000]
[2024-02-11 10:09:25,035: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-11 10:09:35,659: INFO: _internal: 192.168.88.54 - - [11/Feb/2024 10:09:35] "GET / HTTP/1.1" 200 -]
[2024-02-11 10:09:46,908: INFO: _internal: 192.168.88.54 - - [11/Feb/2024 10:09:46] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 10:16:34,487: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.88.54:8000]
[2024-02-11 10:16:34,487: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-11 10:18:59,024: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 10:18:59] "GET / HTTP/1.1" 200 -]
[2024-02-11 10:18:59,117: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 10:18:59] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-02-11 10:19:05,156: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 10:19:05] "[33mPOST /YOUR_SERVER_ENDPOINT HTTP/1.1[0m" 404 -]
[2024-02-11 10:19:12,160: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 10:19:12] "[33mPOST /YOUR_SERVER_ENDPOINT HTTP/1.1[0m" 404 -]
[2024-02-11 10:19:13,076: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 10:19:13] "[33mPOST /YOUR_SERVER_ENDPOINT HTTP/1.1[0m" 404 -]
[2024-02-11 10:22:33,518: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 10:22:33] "[33mPOST /YOUR_SERVER_ENDPOINT HTTP/1.1[0m" 404 -]
[2024-02-11 10:22:34,377: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 10:22:34] "[33mPOST /YOUR_SERVER_ENDPOINT HTTP/1.1[0m" 404 -]
[2024-02-11 10:33:39,418: INFO: _internal: 192.168.88.54 - - [11/Feb/2024 10:33:39] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 10:34:09,776: INFO: _internal: 192.168.88.54 - - [11/Feb/2024 10:34:09] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 10:53:47,783: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 10:53:47] "[33mPOST /YOUR_SERVER_ENDPOINT HTTP/1.1[0m" 404 -]
[2024-02-11 10:53:48,521: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 10:53:48] "[33mPOST /YOUR_SERVER_ENDPOINT HTTP/1.1[0m" 404 -]
[2024-02-11 10:53:49,382: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 10:53:49] "[33mPOST /YOUR_SERVER_ENDPOINT HTTP/1.1[0m" 404 -]
[2024-02-11 10:53:49,874: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 10:53:49] "[33mPOST /YOUR_SERVER_ENDPOINT HTTP/1.1[0m" 404 -]
[2024-02-11 10:59:16,918: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.88.54:8000]
[2024-02-11 10:59:16,918: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-11 10:59:21,652: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 10:59:21] "GET / HTTP/1.1" 200 -]
[2024-02-11 11:00:48,173: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.88.54:5000]
[2024-02-11 11:00:48,173: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-11 11:00:52,487: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:00:52] "GET / HTTP/1.1" 200 -]
[2024-02-11 11:09:27,099: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.88.54:5000]
[2024-02-11 11:09:27,099: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-11 11:09:31,232: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:09:31] "GET / HTTP/1.1" 200 -]
[2024-02-11 11:09:32,338: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:09:32] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-02-11 11:09:37,966: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:09:37] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:09:37,966: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:09:37] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:09:44,675: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:09:44] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:09:44,677: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:09:44] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:09:51,052: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x7fefa0060c10> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-02-11 11:09:51,053: WARNING: polymorphic_function: 6 out of the last 6 calls to <function Model.make_predict_function.<locals>.predict_function at 0x7fefa00861f0> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-02-11 11:09:51,054: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:09:51] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:09:51,055: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:09:51] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:09:53,637: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:09:53] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:09:53,639: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:09:53] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:10:29,996: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:10:29] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:10:30,082: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:10:30] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:10:39,829: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:10:39] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:10:39,833: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:10:39] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:10:41,792: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:10:41] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:10:41,798: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:10:41] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:10:46,166: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:10:46] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:10:46,169: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:10:46] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:10:54,858: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:10:54] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:10:54,862: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:10:54] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:11:01,773: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:11:01] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:11:01,775: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:11:01] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:11:03,523: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:11:03] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:11:03,527: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:11:03] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:11:46,401: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:11:46] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:11:46,523: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:11:46] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:11:57,874: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:11:57] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:11:57,878: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:11:57] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:12:10,240: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:12:10] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:12:10,241: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:12:10] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:12:15,949: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:12:15] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:12:15,949: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:12:15] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:12:17,870: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:12:17] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:12:17,872: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:12:17] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:12:43,868: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:12:43] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:12:43,877: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:12:43] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:12:48,458: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:12:48] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:12:48,460: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:12:48] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:13:12,937: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:13:12] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 11:13:12,939: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 11:13:12] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:09,938: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:09] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:09,939: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:09] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:19,582: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:19] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:19,584: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:19] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:27,811: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:27] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:27,841: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:27] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:38,094: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:38] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:38,094: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:38] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:45,377: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:45] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:45,395: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:45] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:47,620: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:47] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:47,623: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:47] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:49,508: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:49] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:49,509: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:49] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:51,536: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:51] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:51,537: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:51] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:54,477: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:54] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:26:54,479: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:26:54] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:29:02,939: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:29:02] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:29:02,969: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:29:02] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:29:07,576: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:29:07] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:29:07,579: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:29:07] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:29:09,609: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:29:09] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:29:09,617: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:29:09] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:29:14,044: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:29:14] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:29:14,135: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:29:14] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:32:16,711: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:32:16] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 14:32:16,713: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 14:32:16] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:26:58,621: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:26:58] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:26:58,621: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:26:58] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:27:01,580: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:27:01] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:27:01,580: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:27:01] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:38:28,425: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:38:28] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:38:28,427: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:38:28] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:38:36,880: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:38:36] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:38:36,880: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:38:36] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:38:39,554: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:38:39] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:38:39,594: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:38:39] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:38:50,779: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:38:50] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:38:50,801: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:38:50] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:39:43,425: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:39:43] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:39:43,426: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:39:43] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:40:25,972: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:40:25] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:40:26,015: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:40:26] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:42:19,384: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:42:19] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:42:19,386: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:42:19] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:48:03,918: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:48:03] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:48:03,918: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:48:03] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:48:08,566: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:48:08] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:48:08,568: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:48:08] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:48:23,113: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:48:23] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:48:23,151: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:48:23] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:49:08,039: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:49:08] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:49:08,043: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:49:08] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:49:17,580: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:49:17] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:49:17,619: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:49:17] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:59:31,170: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:59:31] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:59:31,175: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:59:31] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:59:41,690: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:59:41] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 15:59:41,691: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 15:59:41] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:00:36,953: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:00:36] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:00:36,960: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:00:36] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:00:39,596: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:00:39] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:00:39,743: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:00:39] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:01:11,924: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:01:11] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:01:11,926: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:01:11] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:01:15,799: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:01:15] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:01:15,799: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:01:15] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:01:22,591: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:01:22] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:01:22,593: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:01:22] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:01:31,837: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:01:31] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:01:31,890: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:01:31] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:01:42,672: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:01:42] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:01:42,681: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:01:42] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:02:45,030: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:02:45] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:02:45,031: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:02:45] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:08:30,844: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:08:30] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:08:30,853: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:08:30] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:08:46,577: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:08:46] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:08:46,578: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:08:46] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:08:53,069: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:08:53] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:08:53,187: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:08:53] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:08:56,610: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:08:56] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:08:56,610: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:08:56] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:26:38,758: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:26:38] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:26:38,760: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:26:38] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:26:47,087: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:26:47] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:26:47,088: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 16:26:47] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 16:42:09,678: INFO: stage_02_prepare_base_model: *******************]
[2024-02-11 16:42:09,678: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-11 16:42:09,680: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 16:42:09,681: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 16:42:09,681: INFO: common: created directory at: artifacts]
[2024-02-11 16:42:09,682: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-11 16:42:09,967: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-11 16:42:10,119: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-11 16:42:13,519: INFO: stage_03_model_training: *******************]
[2024-02-11 16:42:13,519: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-11 16:42:13,521: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 16:42:13,522: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 16:42:13,522: INFO: common: created directory at: artifacts]
[2024-02-11 16:42:13,522: INFO: common: created directory at: artifacts/training]
[2024-02-11 17:37:18,071: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-02-11 17:37:23,431: INFO: stage_04_model_evaluation: *******************]
[2024-02-11 17:37:23,432: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-11 17:37:23,433: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 17:37:23,434: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 17:37:23,434: INFO: common: created directory at: artifacts]
[2024-02-11 17:39:52,725: INFO: common: json file saved at: scores.json]
[2024-02-11 17:39:52,725: INFO: common: json file saved at: scores.json]
[2024-02-11 17:39:55,489: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-11 17:39:55,894: INFO: builder_impl: Assets written to: /tmp/tmpntq0pe25/model/data/model/assets]
[2024-02-11 17:41:41,989: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-11 17:49:17,121: INFO: stage_02_prepare_base_model: *******************]
[2024-02-11 17:49:17,122: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-11 17:49:17,124: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 17:49:17,125: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 17:49:17,126: INFO: common: created directory at: artifacts]
[2024-02-11 17:49:17,126: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-11 17:49:17,474: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-11 17:49:17,622: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-11 17:49:21,041: INFO: stage_03_model_training: *******************]
[2024-02-11 17:49:21,041: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-11 17:49:21,043: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 17:49:21,044: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 17:49:21,044: INFO: common: created directory at: artifacts]
[2024-02-11 17:49:21,044: INFO: common: created directory at: artifacts/training]
[2024-02-11 17:59:18,716: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-11 17:59:18,718: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 17:59:18,719: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 17:59:18,719: INFO: common: created directory at: artifacts]
[2024-02-11 17:59:18,719: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-11 17:59:18,719: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kbJ_Kpp2TqVVXzoy-Ju_QNFJ5tpt6FAu/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-11 18:06:36,773: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1kbJ_Kpp2TqVVXzoy-Ju_QNFJ5tpt6FAu/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-11 18:06:37,517: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-02-11 18:06:44,525: INFO: stage_02_prepare_base_model: *******************]
[2024-02-11 18:06:44,525: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-02-11 18:06:44,527: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 18:06:44,528: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 18:06:44,528: INFO: common: created directory at: artifacts]
[2024-02-11 18:06:44,528: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-11 18:06:44,792: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-11 18:06:44,934: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-02-11 18:06:48,367: INFO: stage_03_model_training: *******************]
[2024-02-11 18:06:48,367: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-02-11 18:06:48,369: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 18:06:48,370: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 18:06:48,370: INFO: common: created directory at: artifacts]
[2024-02-11 18:06:48,370: INFO: common: created directory at: artifacts/training]
[2024-02-11 18:57:31,081: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-02-11 18:57:35,485: INFO: stage_04_model_evaluation: *******************]
[2024-02-11 18:57:35,485: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-02-11 18:57:35,487: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-11 18:57:35,488: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-11 18:57:35,488: INFO: common: created directory at: artifacts]
[2024-02-11 18:59:05,869: INFO: common: json file saved at: scores.json]
[2024-02-11 18:59:05,869: INFO: common: json file saved at: scores.json]
[2024-02-11 18:59:08,629: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-11 18:59:08,996: INFO: builder_impl: Assets written to: /tmp/tmpsjk6dp_c/model/data/model/assets]
[2024-02-11 19:00:48,430: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-02-11 19:43:52,393: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.88.54:5000]
[2024-02-11 19:43:52,393: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-02-11 19:43:54,925: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 19:43:54] "GET / HTTP/1.1" 200 -]
[2024-02-11 19:43:55,046: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 19:43:55] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-02-11 19:44:06,077: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 19:44:06] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 19:44:06,079: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 19:44:06] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 19:44:11,231: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 19:44:11] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 19:44:11,234: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 19:44:11] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 19:44:37,548: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x7f3c9c4f7d30> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-02-11 19:44:37,549: WARNING: polymorphic_function: 6 out of the last 6 calls to <function Model.make_predict_function.<locals>.predict_function at 0x7f3c9c4f7790> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-02-11 19:44:37,550: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 19:44:37] "POST /predict HTTP/1.1" 200 -]
[2024-02-11 19:44:37,550: INFO: _internal: 127.0.0.1 - - [11/Feb/2024 19:44:37] "POST /predict HTTP/1.1" 200 -]
